tests/store/crud/test_create.py

```diff
@@ -25,7 +25,7 @@ def test_add_pending_analysis(raw_analyses: List[dict], store: MockStore, user_e
 
     # THEN it should be stored in the database
     stored_analysis: Analysis = store.get_analysis(
-        case_id=analysis.get("family"),
+        case=analysis.get("family"),
```

Comment:
`case` or `case_name`? Opinions?

Trailblazer does not have a case object so case is technically "free"

Reply:
Intuitively I would see case as an object. Does analysis.get("family") refer to the internal_id of a case (i.e. pet name). If so I would go with case_internal_id even. 

given that trailblazer has close ties with cg we could benefit from keeping naming consistent for ease of transition between repos

Reply:
I would also go for `case_name` or `case_internal_id`